src/app/routing/pages/view-case/view-case.component.ts

```diff
@@ -42,5 +51,12 @@ export class ViewCaseComponent implements OnInit {
             }
         }
     }
+    private setNavigationSections() {
```

Comment:
why not add :void here also :)

Reply:
i can sure

Reply:
Great, so this should be closed once u push update

Reply:
yeap it says outdated... that means it...